src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -4374,6 +4374,53 @@ def test_stencil_region_unsupported(dist_mem):
     assert "Unsupported stencil type 'region' supplied" in \
         str(excinfo.value)
 
+def test_single_stencil_cross2d(dist_mem, tmpdir):
+    ''' Test a single stencil access with an extent value passed from the
+    algorithm layer is treated correctly in the PSy layer. Test both
+    sequential and distributed memory.
+
+    '''
+    _, invoke_info = parse(
+        os.path.join(BASE_PATH, "19.25_single_stencil_cross2d.f90"),
+        api=TEST_API)
+    psy = PSyFactory(TEST_API,
+                     distributed_memory=dist_mem).create(invoke_info)
+    result = str(psy.gen)
+
+    assert LFRicBuild(tmpdir).code_compiles(psy)
+
+    output1 = (
+        "SUBROUTINE invoke_0_testkern_stencil_cross2d_type(f1, f2, f3, f4, "
+        "f2_extent)")
+    assert output1 in result
+    assert "USE stencil_2D_dofmap_mod, ONLY: stencil_2D_dofmap_type\n" in result
```

Comment:
If you are able to merge those two if-blocks I mentioned earlier, this should become a single USE statement.

Reply:
Done